pkg/apis/networking/validation/validation.go

```diff
@@ -81,6 +81,48 @@ func ValidateNetworkPolicySpec(spec *networking.NetworkPolicySpec, fldPath *fiel
 			}
 		}
 	}
+	// Validate egress rules 
+	for i, egress := range spec.Egress {
+		egressPath := fldPath.Child("egress").Index(i)
+		for i, port := range egress.Ports {
```

Comment:
Probably should split out ValidateNetworkPolicyPort and ValidateNetworkPolicyPeer functions to share between ingress and egress validation

Reply:
+1

Reply:
(You "+1"ed this but then didn't actually do it.)